services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/component/DccRevocationListStructureProviderTest.java

```diff
@@ -145,11 +151,17 @@ void shouldCreateCorrectFileStructureFromEmptyDbList() {
   @Test
   void coverFetchDccRevocationList() {
     underTest.fetchDccRevocationList();
+    //final RevocationEtag etag = new RevocationEtag("chunk", dccRevocationClient.getETag());
+    //Optional<List<RevocationEntry>> dccRevocationList = dccRevocationClient.getDccRevocationList();
+    //Mockito.verify(dccRevocationListService, Mockito.times(1)).store(dccRevocationList.get());
+    //Mockito.verify(dccRevocationListService, Mockito.times(1)).store(etag);
```

Comment:
Why is that code commented? I have to admit that I am against adding "dead code". We should either add a TODO with a clear communication as what/when to activate the code, or not add it.

Reply:
Hi, you are right. I will remove this code.